jobs/config.json

```diff
@@ -10310,7 +10309,6 @@
   },
   "pull-test-infra-bazel": {
     "args": [
-      "--affected",
```

Comment:
I think it will cause pr-test-infra to build/test nothing :-) Either leave it for now, or backfill --build and --test?